dowhy/causal_graph.py

```diff
@@ -79,13 +79,14 @@ def __init__(self,
         # Adding node attributes
         self._graph = self.add_node_attributes(observed_node_names)
 
-    def view_graph(self, layout="dot", size=(8, 6), file_name="causal_model"):
-        out_filename = "{}.png".format(file_name)
+    def view_graph(self, layout="dot", size=(8, 6), file_name=None):
```

Comment:
this file also needs "import tempfile" at the top.

Reply:
Oops. Thanks.